### PR TITLE
Pass log level to executed script for more granular logging possibility

### PR DESCRIPTION
**Pull request by andywaltlova**
[HMS-3837](https://issues.redhat.com/browse/HMS-3837)

TBD, for future https://github.com/oamg/convert2rhel-worker-scripts/issues/49

### CODE REVIEW

**Comment by andywaltlova** [comment]
Tested with custom script executable agianst stage today, it works as expected:

- This is log when rhc contains `log-level = "trace"` - `'INFO'` value is passed to script
```txt
...
Apr 03 12:58:35 kvm-02-guest19.rhts.eng.brq.redhat.com rhcd[32119]: 2024/04/03 12:58:35 runner.go:123: Writing temporary script to /var/lib/rhc-worker-script
Apr 03 12:58:35 kvm-02-guest19.rhts.eng.brq.redhat.com rhcd[32119]: 2024/04/03 12:58:35 runner.go:128: Processing script ...
Apr 03 12:58:35 kvm-02-guest19.rhts.eng.brq.redhat.com rhcd[32119]: 2024/04/03 12:58:35 runner.go:131: Executing script...
Apr 03 12:58:35 kvm-02-guest19.rhts.eng.brq.redhat.com rhcd[32119]: 2024/04/03 12:58:35 runner.go:83: Successfully set env variable RHC_WORKER_SCRIPT_MODE
Apr 03 12:58:35 kvm-02-guest19.rhts.eng.brq.redhat.com rhcd[32119]: 2024/04/03 12:58:35 runner.go:90: Successfully set env variable LOG_LEVEL to INFO
Apr 03 12:58:35 kvm-02-guest19.rhts.eng.brq.redhat.com rhcd[32119]: 2024/04/03 12:58:35 runner.go:136: Command is being executed with following env variables:  [YGG_SOCKET_ADDR=unix:@yggd-dispatcher-lDOHNX PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin RHC_WORKER_SCRIPT_MODE=ANALYSIS LOG_LEVEL=INFO]
...

**Comment by andywaltlova** [comment]
And just note, that we have this in our code:

```
		// Yggdrasil < 3.0 does not share its configured log level with the
		// workers in any way
		log.SetLevel(log.LevelInfo)
```

So worker and script will be logging everything as info if rhc doesn't share the log level with us. If/When there is possibility to pass custom variables to rhc workers user will be able to overwrite it.

File: src/runner.go [hunk 1]
```diff
@@ -125,7 +124,12 @@ func processSignedScript(incomingContent []byte) string {
 	// Execute script
 	log.Infoln("Executing script...")
 	cmd := exec.Command(yamlContent.Vars.Interpreter, scriptFileName) //nolint:gosec
-	setEnvVariablesForCommand(cmd, yamlContent.Vars.ContentVars)
+	cmd.Env = os.Environ()
+
+	variables := yamlContent.Vars.ContentVars
+	variables["LOG_LEVEL"] = *config.ScriptLogLevel
+	setEnvVariablesFromYaml(cmd, variables)
+	log.Infoln("Script is being executed with following env variables: ", cmd.Env)
```
Comment (andywaltlova) [inline src/runner.go:132]: Don't know if I should leave this, I initially added it to see what rhc passes down to us, but now I am realizing that we may not want to print it to logs, I don't expect any passwords but we never know

Comment (r0x0d) [reply src/runner.go]: removed!

**Review by r0x0d — APPROVED** [review]

**Review by r0x0d — APPROVED** [review]

**Review by kokesak — APPROVED** [review]

**Pull request merged**